addon/components/file-dropzone/component.js

```diff
@@ -147,7 +147,9 @@ export default Ember.Component.extend({
   didLeaveDropzone(evt) {
     set(this[DATA_TRANSFER], 'dataTransfer', evt.dataTransfer);
```

Comment:
I'm not sure if we should do this if `evt.dataTransfer` is null. Maybe skip this too if `evt.dataTransfer` is null?

Reply:
Yeah, that may be 👍 

I will need to test this- could you provide a clear repro so I can get in and out of this quickly?